docs/userjourneys.md

```diff
@@ -31,10 +31,24 @@ TODO: ACL - mgmt
 
 `clusteradm status` 
 
+### Pivot workflow
+
+`clusteradm config --provider=aws --pivot -o pivot-test.yaml (default --flavor=dev)`
```

Comment:
So I'd really like pivot to be a root level property of the cluster.yaml, and not specific to the tool.  

Thoughts?

Reply:
I think that would fit as well.

So the Cluster object would have:
```yaml
spec:
  pivot: true
```

and then the Cluster reconciler would know what to do with that?

Reply:
Yup.  I'll open an issue in the main ClusterAPI repo and see we can determine the logistics there.